src/app/modules/submission/encrypt-submission/encrypt-submission.utils.ts

```diff
@@ -237,3 +240,26 @@ export const createEncryptedSubmissionDto = (
     version: submissionData.version,
   }
 }
+
+export const getPaymentAmount = (
```

Comment:
it may be better to add a description here, as there are multiple forms of return here

Reply:
Added some description and return type 793abffb1933385ed6cbde30a5a5e92c518922d1